components/list/list-controls.js

```diff
@@ -19,31 +20,47 @@ export class ListControls extends SelectionControls {
 				--d2l-selection-controls-padding: var(--d2l-list-controls-padding, 18px);
 				z-index: 6; /* must be greater than d2l-list-item-active-border */
 			}
+			:host([child-has-color]) {
+				margin: 0 -0.9rem;
+			}
 			:host([no-sticky]) {
 				z-index: auto;
 			}
-			.d2l-list-controls-extend-separator {
-				padding: 0 0.9rem;
+			.d2l-list-controls-color,
+			.d2l-list-controls-extend-separator  {
+				padding: 6px 1.8rem;
 			}
 		`];
 	}
 
 	constructor() {
 		super();
+		this.childHasColor = false;
 		this._extendSeparator = false;
 	}
 
 	connectedCallback() {
 		super.connectedCallback();
 
+		this._selectionControlsInitialPadding = window.getComputedStyle(this).getPropertyValue('--d2l-selection-controls-padding');
+
 		const parent = findComposedAncestor(this.parentNode, node => node && node.tagName === 'D2L-LIST');
 		if (parent) this._extendSeparator = parent.hasAttribute('extend-separators');
 		if (this._extendSeparator) this.style.setProperty('--d2l-selection-controls-padding', '0px');
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+		if (changedProperties.has('childHasColor') && !this._extendSeparator) {
+			if (this.childHasColor) this.style.setProperty('--d2l-selection-controls-padding', '0px');
+			else this.style.setProperty('--d2l-selection-controls-padding', this._selectionControlsInitialPadding);
```

Comment:
This resets to initial padding value if the color is removed and there is not `extend-separators`. Not sure if there is a better way to do this?